qiskit/pulse/library/pulse.py

```diff
@@ -59,29 +61,85 @@ def assign_parameters(self,
         """
         raise NotImplementedError
 
-    @abstractmethod
-    def draw(self, dt: float = 1,
-             style=None,
-             filename: Optional[str] = None,
-             interp_method: Optional[Callable] = None,
-             scale: float = 1, interactive: bool = False,
-             draw_title: bool = False):
+    def draw(self,
```

Comment:
Order of arguments should be maintained for backward compatibility where code input kwargs as args. As a sidenote this is why everyone should use `kwarg=kwarg` when calling a function with a kwarg, to prevent rearrangement of kwargs from breaking code.

After deprecation, these can be dropped.